### PR TITLE
chore(random): deprecate random module

### PR DESCRIPTION
**Pull request by ST-DDT**
We have dedicated modules for pseudo words and actual words, so we should redirect the users to use them instead.

### CODE REVIEW

**Review by ejcheng — DISMISSED** [review]

File: src/modules/random/index.ts [hunk 1]
```diff
@@ -27,14 +27,26 @@ export class RandomModule {
...
+   *
+   * @see faker.lorem.word()
+   * @see faker.word.sample()
    *
    * @example
    * faker.random.word() // 'Seamless'
    *
    * @since 3.1.0
+   *
+   * @deprecated Use `faker.lorem.word()` or `faker.word.sample()` instead.
```
Comment (Shinigami92) [inline src/modules/random/index.ts:44]: Now every single method in this module is deprecated, so we should also annotate the whole module with `@deprecated`

Comment (ST-DDT) [reply src/modules/random/index.ts:44]: Done

**Comment by ST-DDT** [comment]
For some reason, `pnpm run preflight` doesn't fail on lint issues.

**Comment by ST-DDT** [comment]
Is `chore` the correct type? Because chores don't show up in the changelog.

File: src/faker.ts [hunk 2]
```diff
@@ -64,6 +64,10 @@ export class Faker {
   /** @internal */
   private readonly _mersenne: Mersenne = mersenne();
 
+  /**
+   * @deprecated  Use the modules specific to the type of data you want to generate instead.
```
Comment (Shinigami92) [inline src/faker.ts]:
```suggestion
   * @deprecated Use the modules specific to the type of data you want to generate instead.
```

**Review by Shinigami92 — APPROVED** [review]

**Review by ejcheng — APPROVED** [review]

**Pull request merged**